src/bun.js/node/types.zig

```diff
@@ -2117,6 +2118,7 @@ pub const Process = struct {
 
         vm.exit_handler.exit_code = code;
         vm.onExit();
+        if (bun_js.anyUnhandled() and vm.exit_handler.exit_code == 0) vm.exit_handler.exit_code = 1;
```

Comment:
What happens if you run `process.exit()` in `bun test` or in macros? It's uninitialized memory

Reply:
do we allow that?

Reply:
we don't prevent it